src/Header/Header.tsx

```diff
@@ -24,63 +24,138 @@ const Header = () => {
     }
   };
 
-  const linkTextStyle = {
-    visibility: menu ? 'visible' : '',
-    opacity: menu ? '1' : '',
-    transition: 'opacity 200ms',
-    whiteSpace: 'nowrap',
-  } as React.CSSProperties;
+
+
+const width = menu ? '14rem' : '3rem';
+const StyledNav = styled.nav`
+  z-index: 3;
+  height: 100vh;
+  position: fixed;
+  left: 0;
+  background: rgb(24, 21, 21);
+  transition: width 200ms ease;
+  width: ${width};
+
+  ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+`;
+
+const StyledLogo = styled.li`
+  font-weight: bold;
+  text-transform: uppercase;
+  position: relative;
+  top: 0;
+  margin-bottom: 15rem;
+  color: red;
+`;
+
+const StyledNavItem = styled.li`
+  margin-top: auto;
+
+  &:hover {
+    color: white;
+    border-left: solid 3px green;
+  }
+`;
+
+const StyledNavIcon = styled.div`
+  display: flex;
+  align-items: center;
+  height: 3rem;
+  text-decoration: none;
+  cursor: pointer;
+
+  stroke-width: 2;
+  stroke: grey;
+  opacity: 0.5;
+  transition: opacity 100ms;
+
+  &:hover {
+    stroke-width: 2.5;
+    stroke: #fff;
+    opacity: 0.95;
+  }
+`;
+
+const StyledSvg = styled.span`
+  margin: 0 0.35rem;
+
+  &:hover {
+   cursor: pointer;
+  }
+`;
+
+const visibility = menu ? 'visible' : '';
+const opacity = menu ? '1' : '';
+const StyledLinkText = styled.span`
+  color: white;
+  filter: grayscale(100%) opacity(0.7);
+  transition: color 200ms;
+
+  transition: opacity 200ms;
+  whiteSpace: nowrap;
```

Comment:
`whiteSpace` is not a proper css property so it should be kebab cased. Only in proper `jsx` as an object would you used camel case.
```suggestion
  white-space: nowrap;
```